### DISCUSSION TITLE
Unicode Rendering

### DISCUSSION THREAD

**Discussion by yelboudouri** [jgm/pandoc#10309] [General]
Is there a way to convert TeX math notation to a Unicode string? For example, converting `\int x^2 dx` into something like:

```
⌠      
⎮  2   
⎮ x  dx
⌡
```

or for:
```
\begin{bmatrix}
   a & b \\
   c & d
\end{bmatrix}
```
would be rendered like:
```
⌈ a b ⌉
⌊ a b ⌋
```

**Comment by jgm** [comment]
Pandoc will (by default) make an effort to produce a unicode equivalent, but it won't do things like this. (It won't do anything that requires vertical alignment.)
